include/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.hpp

```diff
@@ -92,6 +154,72 @@ class LLA
         const Real& anEllipsoidFlattening
     );
 
+    /// @brief                  Calculate the azimuth angles between two LLA coordinates
+    ///
+    /// @param                  [in] aFirstLLA A first LLA coordinate
+    /// @param                  [in] aSecondLLA A second LLA coordinate
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 Azimuth angles between the two LLA coordinates
+
+    static Pair<Angle, Angle> AzimuthBetween(
+        const LLA& aFirstLLA,
+        const LLA& aSecondLLA,
+        const Length& anEllipsoidEquatorialRadius,
+        const Real& anEllipsoidFlattening
+    );
+
+    /// @brief                  Calculate a point between two LLA coordinates
+    ///
+    /// @param                  [in] aFirstLLA A first LLA coordinate
+    /// @param                  [in] aSecondLLA A second LLA coordinate
+    /// @param                  [in] aRatio A ratio
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 A point between the two LLA coordinates
+
+    static LLA IntermediateBetween(
+        const LLA& aFirstLLA,
+        const LLA& aSecondLLA,
+        const Real& aRatio,
+        const Length& anEllipsoidEquatorialRadius,
+        const Real& anEllipsoidFlattening
+    );
+
+    /// @brief                  Propagate LLA coordinates in provided direction and distance
+    ///
+    /// @param                  [in] aLLA A LLA coordinate
+    /// @param                  [in] aDirection An azimuth angle
+    /// @param                  [in] aDistance A distance
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 Propagated LLA coordinate
+
+    static LLA Forward(
```

Comment:
For the naming, what about
`ForwardLLA`

Reply:
Same as above, IMO the LLA part is obvious based on the function arguments and class.